mrp_bom_attribute_match/reports/mrp_report_bom_structure.py

```diff
@@ -109,3 +109,88 @@ def _get_price(self, bom, factor, product):
                     )
                     price += company.currency_id.round(not_rounded_price)
         return price
+
+    def _get_pdf_line(
+        self,
+        bom_id,
+        product_id=False,
+        qty=1,
+        child_bom_ids=[],  # noqa: B006
```

Comment:
I get that [the original Odoo function](https://github.com/odoo/odoo/blob/f2765d2cab5671a010404c36842bf1b4c4d6350b/addons/mrp/report/mrp_report_bom_structure.py#L213C3-L215C3) is made like that but do we really want to reintroduce a dangerous default?

Reply:
In my personal opinion, changing the signature would be even worse, for me it can stay as it is

Reply:
@ilyasProgrammer what do you say?

Reply:
I would keep it as original.